utils/utils_supported.go

```diff
@@ -0,0 +1,39 @@
+// +build linux darwin
```

Comment:
Drop Darwin, make this utils_linux.go

Reply:
@mheon i cannot do that one yet ... it dives deep into oci and runtime.go respectively.  How about a TODO and gimmie a pass on this one?

Reply:
Alright. I'll request a rename to `utils_unix.go` still, but I'm good to merge once that happens